test/test_sys_proctable_darwin.rb

```diff
@@ -23,11 +23,16 @@ def self.startup
       exec('env', '-i', 'A=B', 'Z=', 'sleep', '60')
     end
 
+    @@pid2 = fork do
+      exec('php', '-d setting=true', '-d directory=/usr/bin', '-R', '\'sleep(120);\'')
+    end
+
```

Comment:
As I mentioned, I think we should try to find a system process that we can use instead. If we do use an external process, I tend to lean towards `Process.spawn` since it gives you more control options.